src/screens/FeedScreen/Header.js

```diff
@@ -259,10 +348,13 @@ const Header = ({
   height,
   source = null,
   headerStyle,
-  onHeaderOptionClicked = () => {},
   showAnonymousOption = false,
-  hideThreeDot,
-  isPostDetail
+  isPostDetail,
+  isFollow = false,
+  onPressFollUnFoll = () => {},
+  onDeletePost = () => {},
+  isShowDelete = false,
+  isSelf = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props have been added to the component to support the new functionalities. It's important to ensure that these props are used correctly throughout the component and that they have appropriate default values if not provided.
```diff
+   showAnonymousOption = false,
+   isPostDetail,
+   isFollow = false,
+   onPressFollUnFoll = () => {},
+   onDeletePost = () => {},
+   isShowDelete = false,
+   isSelf = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->